### PR TITLE
Add getFeatureIdAccessorView for instances

### PR DESCRIPTION
**Pull request by timoore**
Implement getFeatureIdAccessorView for feature IDs stored in the EXT_mesh_gpu_instancing extension. Refactor the existing function, used for accessing feature IDs in primitives, to use common code.

### CODE REVIEW

File: CesiumGltf/src/AccessorUtility.cpp
Comment (j9liu) [inline CesiumGltf/src/AccessorUtility.cpp:65]: Nitpick, but do you mind moving this comment underneath the `case Accessor::ComponentType::UNSIGNED_INT` ?

**Review by j9liu — CHANGES_REQUESTED** [review]
Thanks @timoore! This looks good code-wise, but do you mind doing two things before we merge?

- [ ] Update CHANGES.md
- [ ] Add unit tests

**Comment by timoore** [comment]
@j9liu this awaits your approval!

**Comment by timoore** [comment]
@j9liu thanks for the Changelog entry; ! forgot!

**Comment by j9liu** [comment]
No worries @timoore ! I also wanted to add one more unit test to catch an invalid case, then I'll merge. Will do that in a bit

**Pull request merged**
